pkg/i2gw/providers/kong/converter.go

```diff
@@ -26,7 +26,8 @@ import (
 type converter struct {
 	conf *i2gw.ProviderConf
 
-	featureParsers []i2gw.FeatureParser
+	featureParsers                []i2gw.FeatureParser
+	ImplementationSpecificOptions i2gw.ProviderImplementationSpecificOptions
```

Comment:
correct me if I am wrong but we dont really need this in this struct, right?
we can just pass the function on line 54

Reply:
You're right, although we could say the same thing about `featureParsers` field. As far as I can tell, we only use them on the `converter.ToGatewayAPI` function.

We can either initialize both of them in the constructor or on the `converter.ToGatewayAPI` function. I personally like the constructor option better because it makes the other function easier on the eyes.

Reply:
hmmm you convinced me :) resolving it